crates/re_log_types/src/component_types/transform.rs

```diff
@@ -156,13 +156,15 @@ impl Pinhole {
     ///
     /// [see definition of intrinsic matrix](https://en.wikipedia.org/wiki/Camera_resectioning#Intrinsic_parameters)
     #[inline]
-    pub fn focal_length_in_pixels(&self) -> Vec2D {
-        [self.image_from_cam[0][0], self.image_from_cam[1][1]].into()
+    pub fn focal_length_in_pixels(&self) -> f32 {
+        // Use only the first element of the focal length vector, as we don't support non-square pixels.
+        self.image_from_cam[0][0]
```

Comment:
I think "non-square pixels" is not really descriptive here. Having two different focal lengths for X and Y is common in film: https://en.wikipedia.org/wiki/Anamorphic_format

How common it is in computer vision I don't know, but it seems like this PR is taking us further from supporting it, but looking at all the uses of `focal_length_in_pixels` it seems like it would be trivial to properly support it by just actually returning a `Vec2` here and below